catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
    // something something
}
